R/quantity.R

```diff
@@ -1,11 +1,13 @@
 WITH_DIMENSION_EXTENSION <- "OSPSuite.Core.Domain.WithDimensionExtensions"
+WITH_DISPLAY_UNIT_EXTENSION <- "OSPSuite.Core.Domain.WithDisplayUnitExtensions"
 QUANTITY_EXTENSIONS <- "OSPSuite.Core.Domain.QuantityExtensions"
 
 #' @title Quantity
 #' @docType class
 #' @description  A quantity of the model (with unit, value) such as a Parameter or an Amount
 #' @field value The value of the quantity in unit
-#' @field unit The base unit in which the quantity is defined (Read-Only)
+#' @field unit The base unit in which the quantity value is defined (Read-Only)
+#' @field displayUnit The unit in which the quantity value is usually displayed (Read-Only)

```

Comment:
I know we removed it at some point but I think we need this after all, epsecially for libs which are going to consume stuff in a very generic way such as reporting engine.